server/xpub-server/index.js

```diff
@@ -38,15 +39,30 @@ const registerRoutes = app => {
 const appMessage = action =>
   `${action} Application: ${os.hostname()}, PID: ${process.pid}`
 
-const stopServer = () => {
+const auditApplicationEvent = async message => {
```

Comment:
You probably committed this by mistake - this is to do with the AuditLog changes